src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.figures;
+
+public class Circle extends Figure {
+    private final int radius;
+
+    public Circle(String color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    public int getRadius() {
+        return radius;
+    }
+
+    @Override
+    public double calculateArea() {
+        return (int) Math.PI * radius * radius;

```

Comment:
why are we casting to int?

Reply:
oh, it's because at a certain point of figuring out the task I made all fields double. After some time I was changed that, but for the method "calculateArea" in Circle class, I forgot to fix it.

Reply:
Done!